### ISSUE TITLE
S1075: Delete UT cases that are not relevant to C# 11 raw string literals.

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6381]
EDIT: There are two test cases that have been added [here](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UriShouldNotBeHardcoded.CSharp11.cs#L21) which are not FNs due to raw string literal but due to the fact that the rule does not check this declaration since there is no part of the variable name that matches [the checked variable names set](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.Common/Rules/UriShouldNotBeHardcodedBase.cs#L44).

These FNs are already part of this [rule's test cases](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UriShouldNotBeHardcoded.cs#L44) anyway.
 
### Description

FN for S1075 (UriShouldNotBeHardcoded).

Using C#11 raw string literals, the rule is not raising an issue.

### Repro steps

```csharp
var webChemin = """http://www.mywebsite.com"""; // FN
var windowsChemin = """c:\\blah\\blah\\blah.txt"""; // FN
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
